dockstore-webservice/src/main/resources/migrations.test.samepaths.xml

```diff
@@ -24,6 +24,7 @@
             <column name="defaultversion"/>
             <column name="description"/>
             <column name="email"/>
+            <column name="ischecker" valueBoolean="false"/>
```

Comment:
Why is `ischecker` changing in this PR?

Reply:
Because I originally ran the dockstore primer instructions and had more constraints and issues with it.  Putting this resolves the issues and has no notable downsides.

Reply:
Can you revert this and show us in stand-up what issues you ran into? 
I've tested locally and this should be completely redundant since the default is false anyway.